app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -767,6 +767,10 @@ public static void checkForRisingAllert(Context context) {
             Log.i("NOTIFICATIONS", "checkForRisingAllert: Notifications are currently disabled!!");
             return;
         }
+        if(prefs.getLong("high_alerts_disabled_until", 0) > new Date().getTime()){
```

Comment:
I don't think this is really related to the low high alert. Also this alerts is only fired once so it does not bother too much in any case.